### ISSUE TITLE
NTR, pulmo: Abnormal biochemical composition of bronchoalveolar fluid

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5564]
**Preferred term label:**
Abnormal biochemical composition of broncho-alveolar fluid

**Synonyms**
Abnormal level of proteins in the broncho-alveolar fluid

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal broncho-alveolar lavage fluid morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
-- nanoattribrutions as LMU:katrinknoflach

**Comment by pnrobinson** [comment]
I will add this as 
Abnormal broncho-alveolar fluid protein level for consistency with other terms in HPO.

**Comment by pnrobinson** [comment]
added as HP:0032974
